instrumentation/otel-webserver-module/opentelemetry_module.conf

```diff
@@ -14,6 +14,7 @@ ApacheModuleEnabled ON
 #ApacheModule Otel Exporter details
 ApacheModuleOtelSpanExporter otlp
 ApacheModuleOtelExporterEndpoint collector:4317
+ApacheModuleOtelExporterHeaders api-key=abc123
```

Comment:
Lets keep this commented in the final PR, since the value is for testing

Reply:
Let's discuss what is the format of this ApacheModuleOtelExporterHeaders, should we process by delimiters @DebajitDas @kpratyus 

I just realized the right guide line is "Specifying headers via environment variables"
The OTEL_EXPORTER_OTLP_HEADERS, OTEL_EXPORTER_OTLP_TRACES_HEADERS, OTEL_EXPORTER_OTLP_METRICS_HEADERS environment variables will contain a list of key value pairs, and these are expected to be represented in a format matching to the [W3C Correlation-Context](https://github.com/w3c/baggage/blob/master/baggage/HTTP_HEADER_FORMAT.md), except that additional semi-colon delimited metadata is not supported, i.e.: key1=value1,key2=value2. All attribute values MUST be considered strings.